### ISSUE TITLE
Drush version is 11.3.0-dev, should be 11.3.0?

### ISSUE THREAD

**Issue by gitressa** [drush-ops/drush#5288]
**Describe the bug**
Thanks for the new release! But shouldn't the version be just `11.3.0`?

```
$ drush core:status | grep "Drush version"
Drush version    : 11.3.0-dev      
```

**To Reproduce**
Start a new Drupal 9 project and check the Drush version: 
```
$ composer create-project drupal/recommended-project
$ cd drupal/recommended-project
$ composer require drush/drush
$ drush core:status | grep "Drush version"
Drush version    : 11.3.0-dev
```

**Expected behavior**
Version to be `11.3.0`:
```
$ drush core:status | grep "Drush version"
Drush version    : 11.3.0
```

**Actual behavior**
```
$ drush core:status | grep "Drush version"
Drush version    : 11.3.0-dev
```

**Workaround**
Is there another way to do the desired action?

### System Configuration
| Q               | A
| --------------- | ---
| Drush version  | 11.3.0
| Drupal version | 9.x
| PHP version     | 8.1
| OS             | Linux - Ubuntu 20.04

**Additional information**

**Comment by weitzman** [comment]
Fixed in 11.3.1. Thanks.

**Comment by gitressa** [comment]
Perfect, thanks for a quick fix.
```
$ drush core:status | grep "Drush version"
Drush version    : 11.3.1
```
